perceval/components/core_catalog/controlled_rotation_gates.py

```diff
@@ -92,7 +92,6 @@ class PostProcessedControlledRotationsItem(CatalogItem):
 
     def __init__(self):
         super().__init__("postprocessed controlled gate")
-        self._default_opts['type'] = AsType.PROCESSOR
```

Comment:
I just spotted an error in this file, can you change the l104 `if not "n" in kwargs` into `if "n" not in kwargs` ?